.env.example

```
@@ -0,0 +1,164 @@
+DEBUG=true
+
+# ==============================================
+# Docker Configuration
+# ==============================================
+
+# PostgreSQL (for Docker Compose)
+POSTGRES_PASSWORD=dev_password_123
+POSTGRES_PORT=5432
+
+# Redis (for Docker Compose)
+REDIS_PORT=6379
```

dotenv-linter (3.3.0)

[warning] [DuplicatedKey] The REDIS_PORT key is duplicated

(DuplicatedKey)